tests/yam/validate/validate_extra_kernel_params.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Validate the kernel param of processor.max_cstate=1
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use testapi;
+
+sub run {
+    select_console 'root-console';
+    my $output = script_output('cat /proc/cmdline');
+    my $options = get_var('AGAMA_PROFILE_OPTIONS');
+
+    my ($params_str) = $options =~ /bootloader_extra_kernel_params="([^"]*)"/;
+    my @params = split /,/, $params_str;
+
+    foreach my $param (@params) {
```

Comment:
all the comma-separated values should be in the same order, we could just compare a single string, not split it, right?